lang/c-sharp-pro/ConvexHull.cs

```diff
@@ -0,0 +1,322 @@
+/*
```

Comment:
What's this source file doing here? Should it be in a test directory?

Reply:
You're right. I copied it over from the original c-sharp folder without noticing it shouldn't be here.

Reply:
um, it was in the correct place originally. I just did a wrong copy-paste.

Reply:
both c-sharp and c-sharp-pro now contain the same files:
```
~/ocaml-tree-sitter-semgrep/lang/c-sharp-pro $ git ls-files
Makefile
extensions.txt
fyi.list
projects.txt
test/ok/ConvexHull.cs
```